### ISSUE TITLE
Building pytorch without CuDNN?

### ISSUE THREAD

**Issue by JoshGlue** [pytorch/pytorch#545]
I have an "old" gpu in my laptop (NVIDIA nvs5400m). This gpu has only a cuda compute capability of 2.1. CuDNN requires compute capability > 3.0, so pytorch gpu enabled mode can't run on my gpu. Is it possible to build pytorch without CuDNN, but only with CUDA support? I'm running on Ubuntu 16.04 with CUDA 8.0. If it's not possible, I will just switch to the cpu mode.

**Comment by apaszke** [comment]
Yes, it is possible. cuDNN wrappers automatically switch off if it's not found. You might only see warnings suggesting that you should probably install it, but you can silence or disregard them safely.

**Comment by soumith** [comment]
While the CUDA backend of Torch likely should work with CC 2.1, I dont think we guarantee that everything will work smoothly. For example some of the CUDA kernels we wrote might assume greater register file size than what is available via 2.1. So, expect a bumpy ride.

Saying that, I dont think using nvs5400m will be any more advantageous over using the CPU itself, it's not a fast GPU.
Just wanted to give you a heads-up.

**Comment by JoshGlue** [comment]
Okay, thanks for the information. I'll just give it a try.
